architecture/core_objects/builds.adoc

```diff
@@ -169,24 +170,22 @@ following format:
 
 You can define multiple Docker registry entries in this file. Alternatively, you
 can also add authentication entries to this file by running the `docker login`
-command. The file will be created if it does not exist.
-
-Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
+command. The file will be created if it does not exist. Kubernetes provides
+https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secrets],
+which are used to store your configuration and passwords.
```

Comment:
@soltysh If Kubernetes provides the secret, why are we creating one in the next step? Would it more accurate to say something like "Kubernetes _has the ability to utilize_ secret..." then we go on to create the secret for Kubernetes to utilize?